crates/nu-protocol/src/ast/block.rs

```diff
@@ -22,6 +22,11 @@ impl Block {
     pub fn is_empty(&self) -> bool {
         self.pipelines.is_empty()
     }
+
+    pub fn add_expression(mut self, new: PipelineElement) -> Block {
```

Comment:
What about naming to `push_to_first_pipeline`?

Reply:
Good idea 💡